airbyte-integrations/connectors/source-chartmogul/source_chartmogul/chartmogul.yaml

```diff
@@ -0,0 +1,89 @@
+version: "0.2.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["entries"]
+  requester:
+    url_base: "https://api.chartmogul.com"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  customers_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            page: "{{ (next_page_token['next_page_token'] or 0) + 1 }}"
```

Comment:
is this because the first page is 1?
How about we add an optional `initial_page` to the `PageIncrement` class? It could default to 0 and be set to 1 for chartmogul. I'm sure we'll run into other connectors with this quirk

Reply:
@girarda ok thank you I will wait that CDK update

Reply:
I have updated to use new options of CDK:


@girarda 

```
pagination_strategy:
    type: "PageIncrement"
    start_from_page: 1
```